packages/lesswrong/components/form-components/LocationFormComponent.tsx

```diff
@@ -111,14 +111,24 @@ const LocationFormComponent = ({document, updateCurrentValues, classes}: {
   classes: ClassesType,
 }) => {
   const location = document?.location || ""
-  const [ mapsLoaded ] = useGoogleMaps("LocationFormComponent")
+  const [ mapsLoaded ] = useGoogleMaps("CommunityHome")
```

Comment:
I saw an error in the console regarding including the Google Maps API multiple times on the page, so I updated all the places where we call `useGoogleMaps` to use the same identifier to avoid that. I'm not sure if there is a good reason for them to have different identifiers?

Reply:
This seems correct to me 🙂 I think you would want to use a different identifier string if the optional `libraries` parameter of `useGoogleMaps` was different, but otherwise yeah I think they should be the same.